### PR TITLE
AVX-87 create AP_Periph 1.0.0 release for CAN TXR ADSB and radar

### PR DESCRIPTION
**Pull request by rabad-matternet**
- updated version
- updated AP_Periph readme

### CODE REVIEW

**Review by AydenChubbic — APPROVED** [review]

**Review by adriennet823 — APPROVED** [review]

**Review by bhiln — APPROVED** [review]

File: Tools/AP_Periph/README.md [hunk 1]
```diff
@@ -31,7 +31,12 @@ More can be added using the hwdef.dat system
    - Sync'ed to Arudpilot/ardupilot (master) at hash b0fa5fa
    - Current CAN TXR AP_Periph projects supported are:
      - f303-ADSB
-     - f303-USD1  
+     - f303-USD1
+
+## Mattern AP_Periph Release
```
Comment (zalice) [inline Tools/AP_Periph/README.md]: Mattern --> Matternet

File: Tools/AP_Periph/version.h [hunk 2]
```diff
@@ -3,10 +3,10 @@
 #define THISFIRMWARE "AP_Periph V1.3dev"
 
 // the following line is parsed by the autotest scripts
-#define FIRMWARE_VERSION 1,3,0,FIRMWARE_VERSION_TYPE_DEV
+#define FIRMWARE_VERSION 1,0,0,FIRMWARE_VERSION_TYPE_DEV
```
Comment (zalice) [inline Tools/AP_Periph/version.h]: What is the motivation for not tracking with the mainstream AP_Periph version/?

**Comment by zalice** [comment]
Are two separate builds needed for ADSB and radar? 1 for each? Or is the same binary flashed for both use cases?

File: Tools/AP_Periph/version.h (hunk 2 above)
Comment (rabad-matternet) [reply Tools/AP_Periph/version.h]: I guess we could.
This is the version that shows up in uavcan_gui_tool.
The intent was to use this to track our CAN TXR firmware version.

File: Tools/AP_Periph/README.md (hunk 1 above)
Comment (rabad-matternet) [reply Tools/AP_Periph/README.md]: Done!

File: Tools/AP_Periph/version.h (hunk 2 above)
Comment (rabad-matternet) [reply Tools/AP_Periph/version.h:6]: > Are two separate builds needed for ADSB and radar? 1 for each? Or is the same binary flashed for both use cases?

@zalice There will be separate binaries for ADSB and radar (though technically the bootloader is the same).  But they will be built from the same code.

**Comment by rabad-matternet** [comment]
> Are two separate builds needed for ADSB and radar? 1 for each? Or is the same binary flashed for both use cases?

@zalice There will be separate binaries for ADSB and radar (though technically the bootloader is the same). But they will be built from the same code.

File: Tools/AP_Periph/README.md [hunk 3]
```diff
@@ -31,7 +31,12 @@ More can be added using the hwdef.dat system
    - Sync'ed to Arudpilot/ardupilot (master) at hash b0fa5fa
```
Comment (rabad-matternet) [inline Tools/AP_Periph/README.md:31]: @zalice This is where we can trace back to Arudupilot (master)

File: Tools/AP_Periph/version.h (hunk 2 above)
Comment (rabad-matternet) [reply Tools/AP_Periph/version.h]: As discussed offline, this change has been reverted

**Pull request merged**